cmd/cloud/group.go

```diff
@@ -21,6 +22,7 @@ func init() {
 	groupUpdateCmd.Flags().String("description", "", "An optional description for this group of installations.")
 	groupUpdateCmd.Flags().String("version", "", "The Mattermost version for installations in this group to target.")
 	groupUpdateCmd.Flags().String("image", "mattermost/mattermost-enterprise-edition", "The Mattermost Container image to use.")
+	groupUpdateCmd.Flags().Int64("max-rolling", 0, "The maximum number of installations that can be updated at one time when a group is updated")
```

Comment:
why is zero here?

Reply:
Ah, I needed a value here that basically meant "don't change the original value". The value `1` isn't good because you may want to actually set max rolling to 1.

I can do a follow-up PR if we find this confusing going forward.